spec-0000/index.md

```diff
@@ -70,6 +70,15 @@ The situation has since improved due to improved installations via binary wheels
 
 ### Drop Schedule
 
+Below is an auto generated schedule, with recommended date for considering
+stopping support. We suggest that the next release in a given quarter is
```

Comment:
The grammar might be off here? How about "..., with recommended dates at which dropping support should be considered." Not an expert either tough.